applications/MappingApplication/custom_utilities/interface_geometry_object.h

```diff
@@ -389,6 +389,16 @@ class InterfaceGeometryObject : public InterfaceObject
     ///@name Private  Access
     ///@{
 
+        
+    ///@}
+    ///@name Serialization
+    ///@{
+
+    friend class Serializer;
+    
+    virtual void save(Serializer& rSerializer) const {}
```

Comment:
You should call the base class `save` class as described in the [old wiki](http://kratos-wiki.cimne.upc.edu/index.php/How_to_use_Serialization)